process/support_team.md

```diff
@@ -0,0 +1,93 @@
+# The Made Tech Support Team
+
+## High level overview
+
+Our Support Team is best thought of as the 'on-demand' equivalent to our AaaS offering. They undertake smaller tactical pieces of work which normally would necessitate the spinning up of a dedicated Sprint team. The length of work undertaken varies from less than an hour, up to about a month. Beyond that our AaaS offering becomes more appropriate.
+
+The Support Team is run by Nick Wood, who oversees day to day operations, and manages relationships with our support customers.  
+
+## Support Team Structure
+
+The support team doesn't have a fixed roster of staff and instead consists of a number of developers based availibility of staff and customer demand for support work. The support team is formed of:
+ * The **'Triage Team'**, a core rota of staff each contributing a day or more of their week to oversee intake of new issues, and fixing small and critical issues.
+ * A variable number of **'Support Pairs'**. These pairs are stable from week to week, and generally have an overarching goal which spans many weeks.
+
+## Support issue lifecycle
+
+All support issues begin life as an email into support@madetech.com. These are forwarded automatically into our GrooveHQ inbox to be sorted by the Triage Team. We aim to resolve all new issues within 48 hours with one of the following resolutions:
```

Comment:
both work, but .com is preferred (due to email routing causing duplicates if .co.uk is used)